packages/paste-core/components/topbar/src/Topbar.tsx

```diff
@@ -6,36 +6,41 @@ import type {BoxProps} from '@twilio-paste/box';
 export interface TopbarProps extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
   element?: BoxProps['element'];
+  topbarSkipLinkID: string;
 }
 
-export const Topbar = React.forwardRef<HTMLDivElement, TopbarProps>(({element = 'TOPBAR', children, ...props}, ref) => {
-  return (
-    <Box
-      {...safelySpreadBoxProps(props)}
-      ref={ref}
-      element={element}
-      width="100%"
-      minHeight="sizeTopbar"
-      backgroundColor="colorBackgroundWeak"
-      borderBottomWidth="borderWidth10"
-      borderBottomStyle="solid"
-      borderBottomColor="colorBorderWeaker"
-      paddingX="space70"
-      paddingY="space60"
-      position="sticky"
-      top="0"
-      display="flex"
-      justifyContent="space-between"
-      alignItems="center"
-      zIndex="zIndex10"
-    >
-      {children}
-    </Box>
-  );
-});
+export const Topbar = React.forwardRef<HTMLDivElement, TopbarProps>(
+  ({element = 'TOPBAR', topbarSkipLinkID, children, ...props}, ref) => {
```

Comment:
question: I was thinking the prop would just be `id: string` (required), did you name it this way so people knew there was a relationship here?

Reply:
That was the theory, to be more explicit on why we need it. My initial fear was that if it's just `id`, when we require it with typescript, there's nothing really telling the person that its anything related to the Sidebar and skip links. It's just "giev me an id"

We could TSDoc it on the property to give a VS Code hint, but requires the person to read the popup.